torch/csrc/api/include/torch/nn/modules/activation.h

```diff
@@ -120,5 +120,28 @@ class TORCH_API LogSigmoidImpl : public torch::nn::Cloneable<LogSigmoidImpl> {
 
 TORCH_MODULE(LogSigmoid);
 
+
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Softmax ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+/// Applies the Softmax function.
+/// See https://pytorch.org/docs/master/nn.html#torch.nn.Softmax to learn
+/// about the exact behavior of this module.
+class TORCH_API SoftmaxImpl : public Cloneable<SoftmaxImpl> {
+ public:
+  SoftmaxImpl(const SoftmaxOptions& options_);
```

Comment:
Please add default constructor, like in all other modules

Reply:
Added default constructor.